week1/Task2/Program.cs

```diff
@@ -0,0 +1,21 @@
+﻿string input = "Intellectualization";
+string inputToLowerCase = input.ToLower();
+
+string Vowels= "aeiou";
+char[] VowelsArray = Vowels.ToCharArray();
+
+int GetVowelCount(string inputToLowerCase){
+    int VowelsCount = 0;
+
+    foreach (char letter in inputToLowerCase)
+    {
+        foreach (char vowel in VowelsArray)
```

Comment:
There are some other efficient options.

- Instead of second "foreach" for comparing the chars. you can write it:
     if (Vowels.Contains(letter))
           VowelsCount++;

- Or one way is to have a single if statement: if (c == 'a' || c == 'e' || c == 'i' || c == 'o' || c == 'u')

Reply:
great, done